src/styles/index.scss

```diff
@@ -1,3 +1,22 @@
+@import './utils/variables';
+@import 'blocks/header';
+@import 'blocks/grid';
+@import 'blocks/card';
+@import 'blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
```

Comment:
The body hasn't a padding
```suggestion
```

Reply:
@volodymyr-soltys97 have run deploy again, the demo has styles now